packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -18,6 +18,61 @@ export type Options = [Config];
 
 export type MessageIds = 'unbound';
 
+const nativelyBoundMembers = [
+  ...[
+    Promise,
+    BigInt,
+    Number,
+    Object,
+    String,
+    RegExp,
+    Symbol,
+    Array,
+    Proxy,
+    Date,
+  ].map(x => [x.name, x]),
+  ['Infinity', Infinity],
```

Comment:
nit - `Infinity` has no methods (or properties)

Reply:
for some reason I saw `toPrecision` and thought it did :joy: